R/track-util.R

```diff
@@ -0,0 +1,142 @@
+#' track helpers
```

Comment:
An additional benefit from moving the functions into track.R: you resolve the `# 'export` issues.